src/fides/api/service/connectors/scylla_query_config.py

```diff
@@ -0,0 +1,92 @@
+import re
+from typing import Any, Dict, List, Optional, Tuple
+
+from fides.api.graph.config import Field
+from fides.api.models.policy import Policy
+from fides.api.service.connectors.query_config import SQLLikeQueryConfig
+
+ScyllaDBStatement = Tuple[str, Dict[str, Any]]
+"""
+A ScyllaDB statement is a tuple whose first component is the query string
+and whose second component is a dict of parameters to pass in to the query.
+
+E.g
+('SELECT name FROM users WHERE user.id = %(user_id)s', { "user_id": 123 })
+"""
+
+
+class ScyllaDBQueryConfig(SQLLikeQueryConfig[ScyllaDBStatement]):
+    def query_to_str(
+        self, t: ScyllaDBStatement, input_data: Dict[str, List[Any]]
+    ) -> str:
+        """string representation of a query for logging/dry-run"""
+
+        def transform_param(p: Any) -> str:
+            if isinstance(p, str):
+                return f"'{p}'"
+            return str(p)
+
+        query_str = str(t[0])
```

Comment:
oof yeah, this code's looking a bit rough haha tbh I just copied a lot of it from the SQL implementation and made the Scylla-related changes . I'll improve it

> Do we need to cast this to a string, since the type hint for ScyllaDBStatement already specifies this as a string?

You're right, I don't think we need the cast